src/components/delete-account/tests/delete-account-controller.test.ts

```diff
@@ -55,59 +55,7 @@ describe("delete account controller", () => {
   });
 
   describe("deleteAccountPost", () => {
-    describe("when not supporting DELETE_SERVICE_STORE", () => {
```

Comment:
I think this makes the case @divyaparameswaran and @DougsCER were arguing for about removing these.

These tests become so much simpler with less repetition!